setup-scripts/blocklist-airtable.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+import traceback, os, re, asyncio, requests, json, discord
+from bot_utils import setup, send_msg
+
+bot_token = setup()
+client = discord.Client()
+
+AIRTABLE_API_KEY = os.getenv("AIRTABLE_API_KEY")
+AIRTABLE_BASE = os.getenv("AIRTABLE_BASE", "app89plJvA9EqTJEc")
+AIRTABLE_TABLE = os.getenv("AIRTABLE_TABLE", "Table%201")
+AIRTABLE_FIELD = os.getenv("AIRTABLE_FIELD", "Link")
```

Comment:
are any of those defaults sensitive and should be removed from the repo?

Reply:
Nope, those are fine, only the `AIRTABLE_API_KEY` is sensitive